demos/noise_suppression_demo/python/README.md

```diff
@@ -0,0 +1,72 @@
+# Noise Suppression Python\* Demo
+
+This README describes the Noise Suppresion demo application.
+
+## How It Works
+
+Upon the start-up the demo application reads command line parameters and loads a network to Inference engine.
+It also read user-provided sound file with mix of speech and some noise to feed it into the network by small sequential patches.
+The output of network is also sequence of audio patches with clean speech. The patches collected together and save into ouput audio file.
+
+## Preparing to Run
+
+The list of models supported by the demo is in <omz_dir>/demos/noise_suppression_demo/python/models.lst file.
+This file can be used as a parameter for [Model Downloader](../../../tools/downloader/README.md) and Converter to download and, if necessary, convert models to OpenVINO Inference Engine format (\*.xml + \*.bin).
+
+### Supported Models
+
+* noise-suppression-poconetlike-0001
+
+> **NOTE**: Refer to the tables [Intel's Pre-Trained Models Device Support](../../../models/intel/device_support.md) and [Public Pre-Trained Models Device Support](../../../models/public/device_support.md) for the details on models inference support at different devices.
+## Running
+
+Running the application with the `-h` option yields the following usage message:
+```
+python3 noise_suppression.py -h
+```
+The command yields the following usage message:
+```
+usage: noise_suppression.py [-h] -m MODEL -i INPUT [-o OUTPUT] [-d DEVICE]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -m MODEL, --model MODEL
+                        Required. Path to an .xml file with a trained model
+  -i INPUT, --input INPUT
+                        Required. Path to a 16kHz sound file with speech+noise
+  -o OUTPUT, --output OUTPUT
+                        Optional. Path to output sound file with speech
+  -d DEVICE, --device DEVICE
+                        Optional. Target device to perform inference
+                        on. Default value is CPU
+```
+
+You can use the following command to try the demo (assuming the model from the Open Model Zoo, downloaded with the
+[Model Downloader](../../../tools/downloader/README.md) executed with "--name dns*"):
+```
+    python3 noise_suppression.py
+            --model=<path_to_model>/noise-suppression-poconetlike-0001.xml
+            --input=noisy.wav
+            --output=cleaned.wav
```

Comment:
These are 4 commands. You need to write them it in one line or add `\`.
Please remove 4 spaces in front.